docs/docs-content/integrations/microk8s.md

```diff
@@ -10,91 +10,275 @@ logoUrl: "https://registry.spectrocloud.com/v1/kubernetes-microk8s/blobs/sha256:
 tags: ["packs", "microk8s", "kubernetes"]
 ---
 
-MicroK8s is a Cloud Native Computing Foundation (CNCF) certified upstream Kubernetes deployment that runs entirely on
-your workstation or edge device. It runs all Kubernetes services natively without virtual machines and packs all the
-required libraries and binaries.
+[MicroK8s](https://microk8s.io) is a lightweight Cloud Native Computing Foundation (CNCF) certified Kubernetes
+distribution developed by [Canonical](https://canonical.com). It deploys all Kubernetes services in a single, fully
+contained package, while also offering out-of-the-box [add-ons](https://microk8s.io/docs/addons).
+
+Palette offers MicroK8s as an available Kubernetes distribution for creating clusters. You can use MicroK8s as the
+Kubernetes layer when creating a [cluster profile](../profiles/profiles.md) and then use the cluster profile to deploy a
+cluster.
 
 ### Support Lifecycle
 
-We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by
-the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. Refer to the
+We support other Kubernetes distributions, such as K3s and RKE2, until their official End-of-Life (EOL). The EOL is set
```

Comment:
That sentence sounds a bit weird in this context. We discuss about MicroK8s, but we mention other distros EOL.

Reply:
I agree and rewrote the sentence :)